lib/plug/request_id.ex

```diff
@@ -34,6 +34,12 @@ defmodule Plug.RequestId do
 
           plug Plug.RequestId, http_header: "custom-request-id"
 
+    * `:assign_as` - The name of the key that will be used to store the
+      discovered or generated request id in `conn.private`. If not provided,
```

Comment:
Given the option is called `assign_as`, I think we should put it in assigns. :)

Reply:
Sounds good to me.